frontend/cypress/unit/stores/entityDetails/EntityDetailsStore.test.ts

```diff
@@ -0,0 +1,39 @@
+import { id1, dummies } from '../../../fixtures/entityDetails/entityDetails';
+import EntityDetails from '../../../../src/stores/details/EntityDetailsStore';
+import { createContainer } from '../../../../src/dependency-injection/DependencyInjectionContext';
+import EntityDetailsState from '../../../../src/stores/details/EntityDetailsStateStore';
+
+describe('EntityDetailsStore', () => {
+  // Global setup
+  let entityDetails: EntityDetails;
+  let entityDetailsState: EntityDetailsState;
+  const container = createContainer();
+
+  beforeEach(() => {
+    entityDetails = container.get(EntityDetails);
+    entityDetailsState = container.get(EntityDetailsState);
+  });
+
+  describe('Details store', () => {
+    it('should initially return null', () => {
+      // Act
+      const actual = entityDetails.getValue();
+      const expected = null;
+
+      // Assert
+      expect(actual).to.be.eq(expected);
+    });
+    it('should return details of the node with id 1', () => {
+      // Arrange
+      entityDetails = container.get(EntityDetails);
+      cy.intercept('http://localhost:8080/api/getNodesById*', id1);
+      entityDetailsState.setState({ node: 1, edge: null });
+
+      // Act
+      const actual = entityDetails.getValue();
```

Comment:
Does anyone know why `actual` is `null` here, after updating the StateStore in line 30?

Reply:
I think the intercept is not working properly

Reply:
Think you're right. `intercept` doesn't seem to intercept requests from within the container. But I need the container for DI :(

Reply:
You could overwrite the service with a mock service. There, return a static value, something like that:

```ts
class MockXyzService extends XyzService {
    getNodesById(...) { // This method is overwritten
        return of(mockValue);
    }
}
```

Reply:
Thank you! Got it